nexus/scripts/build/Dockerfile

```diff
@@ -1,6 +1,4 @@
-ARG PYTHON_VERSION=3.9
-ARG PLATFORM=amd64
-FROM --platform=linux/${PLATFORM} python:${PYTHON_VERSION}
+FROM nvidia/cuda:12.2.0-runtime-ubuntu20.04
```

Comment:
lol it actually worked

Reply:
i know haha. gotta figure out what is missing/wrong in the original image